packages/components/src/button/tests/chromatic/ButtonGroup.chroma.jsx

```diff
@@ -3,165 +3,112 @@ import { Button, ButtonGroup, IconButton } from "@components/button";
-        .build();
-}
+export default {
+    component: ButtonGroup,
+    title: "Chromatic/ButtonGroup"
+};
 
-stories()
-    .add("default", () =>
-        <Stack>
-            <ButtonGroup size="sm">
-                <Button variant="secondary">Cutoff</Button>
-                <Button variant="secondary">Cutoff</Button>
-                <Button variant="secondary">Cutoff</Button>
-            </ButtonGroup>
-            <ButtonGroup>
-                <Button variant="secondary">Cutoff</Button>
-                <Button variant="secondary">Cutoff</Button>
-                <Button variant="secondary">Cutoff</Button>
-            </ButtonGroup>
-        </Stack>
-    )
-    .add("icon button", () =>
-        <Inline gap={13}>
-            <Stack>
-                <ButtonGroup size="sm">
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                </ButtonGroup>
-                <ButtonGroup>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                </ButtonGroup>
-            </Stack>
-            <Stack>
-                <ButtonGroup size="sm">
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                </ButtonGroup>
-                <ButtonGroup>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                    <IconButton variant="secondary" aria-label="Add"><AddIcon /></IconButton>
-                </ButtonGroup>
-            </Stack>
-        </Inline>
-    )
-    .add("fluid", () =>
-        <ButtonGroup fluid>
+export const Default = () =>
+    <Stack>
+        <ButtonGroup size="sm">
             <Button variant="secondary">Cutoff</Button>
             <Button variant="secondary">Cutoff</Button>
             <Button variant="secondary">Cutoff</Button>
         </ButtonGroup>
-    )
-    .add("disabled", () =>
-        <ButtonGroup disabled>
+        <ButtonGroup>
             <Button variant="secondary">Cutoff</Button>
             <Button variant="secondary">Cutoff</Button>
             <Button variant="secondary">Cutoff</Button>
         </ButtonGroup>
-    )
-    .add("align", () =>
+    </Stack>;
+
+Default.storyName = "default";
+
+export const ButtonGroupIconButton = () =>
```

Comment:
why is this prefixed with ButtonGroup?

Reply:
Because IconButton is imported, it's an Orbit component